presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveCsvStorageFormat.java

```diff
@@ -0,0 +1,176 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.prestosql.tests.hive;
+
+import io.prestosql.tempto.ProductTest;
+import io.prestosql.tempto.assertions.QueryAssert.Row;
+import io.prestosql.tempto.query.QueryResult;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static com.google.common.collect.ImmutableList.toImmutableList;
+import static io.prestosql.tempto.assertions.QueryAssert.Row.row;
+import static io.prestosql.tempto.assertions.QueryAssert.assertThat;
+import static io.prestosql.tempto.query.QueryExecutor.query;
+import static io.prestosql.tests.TestGroups.STORAGE_FORMATS;
+import static java.lang.String.format;
+
+public class TestHiveCsvStorageFormat
+        extends ProductTest
+{
+    private static final String TPCH_SCHEMA = "tiny";
+
+    @Test(groups = STORAGE_FORMATS)
```

Comment:
this triggers intellij warning for me
please consider using `@Test(groups = {STORAGE_FORMATS})` as a workaround

Reply:
what warnning do you get? I didn't get any.